webui/react/Makefile

```diff
@@ -104,7 +104,7 @@ test:
 
```

Comment:
**chore:** the `test` target above just needs a slight update to work w/ vitest (non-watch mode):
```
npm run test -- run
```